www/include/core/footer/footerPart.php

```diff
@@ -33,307 +33,502 @@
-                        'margin-bottom': '0px'
-                    });
-                    if (document.exitFullscreen) {
-                        document.exitFullscreen();
-                    } else if (document.msExitFullscreen) {
-                        document.msExitFullscreen();
-                    } else if (document.mozCancelFullScreen) {
-                        document.mozCancelFullScreen();
-                    } else if (document.webkitExitFullscreen) {
-                        document.webkitExitFullscreen();
-                    }
-                } else {
-                    jQuery(
-                      "#actionBar, .pathWayBracket, .imgPathWay, .pathWay, hr, #QuickSearch, #menu1_bgcolor," +
-                      " #footer, #menu_1, #Tmenu , #menu_2, #menu_3, #header, .toHideInFullscreen"
-                    ).addClass('tohide');
-                    jQuery("#fullscreenIcon").attr("src", "./img/icons/fullscreen_off.png");
-                    jQuery('#contener').css({
-                        'height': '100%'
-                    });
-                    jQuery('#Tcontener').css({
-                        'margin-bottom': '0px'
-                    });
-                    // go full-screen
-                    if (i.requestFullscreen) {
-                        i.requestFullscreen();
-                    } else if (i.webkitRequestFullscreen) {
-                        i.webkitRequestFullscreen();
-                    } else if (i.mozRequestFullScreen) {
-                        i.mozRequestFullScreen();
-                    } else if (i.msRequestFullscreen) {
-                        i.msRequestFullscreen();
-                    }
-                }
+/**
+ * Getting Centreon UUID.
+ */
+$centreonUUID = new CentreonUUID($pearDB);
+$uuid = $centreonUUID->getUUID();
+
+/**
+ * Getting Platform statistics.
+ */
+$output = new ConsoleOutput();
+$logger = new ConsoleLogger($output);
+$centreonStats = new CentreonStatistics($logger);
+$stats = $centreonStats->getPlatformInfo();
+
+/**
+ * Getting License informations.
+ */
+$productLicense = 'Open Source';
+$licenseClientName = null;
+try {
+    $centreonModules = ['epp', 'bam', 'map', 'mbi'];
+    $licenseObject = $dependencyInjector['lm.license'];
+    $licenseInformation = [];
+    foreach ($centreonModules as $module) {
+        $licenseObject->setProduct($module);
+        $isLicenseValid = $licenseObject->validate(false);
+        if ($isLicenseValid && !empty($licenseObject->getData())) {
+            $licenseInformation[$module] = $licenseObject->getData();
+            $licenseClientName = $licenseInformation[$module]['client']['name'];
+            if ($module === 'epp') {
+                $productLicense = 'IT Edition';
+            }
+            if (in_array($module, ['mbi', 'bam', 'map'])) {
+                $productLicense = 'Business Edition';
             }
         }
+    }
+} catch (\Exception $ex) {
+    error_log($ex->getMessage());
```

Comment:
Could you use Monolog instead ?

Reply:
it is a little bit overwhelming to use Monolog here.